sklearn/linear_model/stochastic_gradient.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, loss, penalty='l2', alpha=0.0001, C=1.0,
                  l1_ratio=0.15, fit_intercept=True, max_iter=None, tol=None,
                  shuffle=True, verbose=0, epsilon=0.1, random_state=None,
                  learning_rate="optimal", eta0=0.0, power_t=0.5,
-                 warm_start=False, average=False, n_iter=None):
+                 warm_start=False, average=False, n_iter=None, set_future_warning=True):
```

Comment:
I don't think we want this as a constructor parameter. It should be a parameter to `_validate_params`

Reply:
Thanks. I'll modify the code.